unlock-app/src/theme/media.ts

```diff
@@ -1,7 +1,7 @@
 import styled, { css } from 'styled-components'
 import { MAX_DEVICE_WIDTHS } from '../constants'
 
-let sizes = {
+let sizes: { [key: string]: any } = {
```

Comment:
this is because they can be either string or number, right? Is there any chance of using a union type like `sizes: { [key: string]: number | string }`?

Reply:
Actually, they can be number or boolean. However, with that `number | boolean` type in there, the code that generates the Media object starts to error and I decided that it wasn't worth the time to go further.